buildLinuxDist.sh

```diff
@@ -1,31 +1,33 @@
 #!/bin/sh
```

Comment:
I can do that for this one. We only need the linux script because these build steps need to run from within the docker build/OS, the other platforms are all handled from within their respective github workflows, so I believe the others can/should be deleted.